.circleci/config.yml

```diff
@@ -161,9 +161,9 @@ jobs:
             fi
             export STORE_JAEGER_TRACES=true
             pushd test
-            CGO_ENABLED=0 GOOS=linux go build -o ../cloudtest ./cloudtest/cmd/cloudtest
+            go get github.com/networkservicemesh/cloudtest
```

Comment:
Should we add tag for cloudtest tool?

Reply:
I think we could yes, but we will need to mark cloudtest as release :-)